src/commands/synthetics/run-tests-lib.ts

```diff
@@ -192,52 +168,117 @@ const getTestListBySearchQuery = async (
   }))
 }
 
-export const getTestsList = async (
+export const getFinalVersionOfTestsFromCi = async (
```

Comment:
I appreciate the specificity, but why `FromCi` here? Maybe a `getFinalVersionOfTests` would be sufficient?

Reply:
Hmm i had the `FromCi` bit in there since later on we get more configuration information from the test details in Datadog. So after this function the test won't be changed by configuration from the ci anymore but they will be enriched some more from datadog later on 🤔 don't know if that distinction is enough to keep the `FromCi` though

Reply:
I would lean toward the wording `TestOverride` rather than `FromCI`: this is the synthetics `RunTestsLib` which could be called outside of CI, strictly speaking.

Reply:
In the end went with `getTriggerConfigs` as the result of this function is referred as `triggerConfigs` later one also it describes well what the function does